src/test/resources/org/prebid/server/it/openrtb2/adf/test-adf-bid-request.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "tid",
+  "imp": [
+    {
+      "id": "impId001",
+      "native": {
+        "request": "{\"ver\":\"1.2\",\"context\":1,\"plcmttype\":4,\"plcmtcnt\":1,\"assets\":[{\"id\":2,\"required\":1,\"title\":{\"len\":90}},{\"id\":6,\"required\":1,\"img\":{\"type\":3,\"wmin\":128,\"hmin\":128,\"mimes\":[\"image/jpg\",\"image/jpeg\",\"image/png\"]}},{\"id\":7,\"required\":1,\"data\":{\"type\":2,\"len\":120}}]}",
+        "ver": "1.2"
+      },
+      "tagid": "1",
+      "ext": {
+        "bidder": {
+          "mid": "1"
```

Comment:
Let's remove tagId in auction request, and check that masterId is being added as tagid